### PR TITLE
ShapeNet Dataset

### PR DESCRIPTION
**Pull request by JerryJiehanWang**
This is a PR cloned directly from: https://github.com/NVIDIAGameWorks/kaolin/pull/345

### CODE REVIEW

File: kaolin/io/obj.py [hunk 1]
```diff
@@ -16,37 +16,45 @@
 import warnings
 from collections import namedtuple
 
-from PIL import Image
 import numpy as np
 import torch
+from PIL import Image
+
+ObjMesh = namedtuple('ObjMesh',
```
Comment (Caenorst) [inline kaolin/io/obj.py]: Please don't rename the nametuple

File: kaolin/io/__init__.py [hunk 2]
```diff
@@ -2,5 +2,9 @@
 from . import materials
 from . import obj
 from . import off
+<<<<<<< HEAD
```
Comment (Caenorst) [inline kaolin/io/__init__.py]: merge conflicts?

File: kaolin/io/obj.py (hunk 1 above)
Comment (Caenorst) [reply kaolin/io/obj.py]: I think the string should also be renamed

**Comment by Caenorst** [comment]
some commits are not signed apparently

File: kaolin/io/obj.py (hunk 1 above)
Comment (JerryJiehanWang) [reply kaolin/io/obj.py]: changed.

File: tests/python/kaolin/io/test_shapenet.py [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+import os
+
+import pytest
+import torch
+
+from kaolin.io.obj import return_type
+from kaolin.io.shapenet import ShapeNet
+
+SHAPENET_PATH = 'data/ShapeNet'
```
Comment (Caenorst) [inline tests/python/kaolin/io/test_shapenet.py]: I think we should assume `/data/ShapeNet`, `data/ShapeNet` is a relative path and should avoided.

Comment (JerryJiehanWang) [reply tests/python/kaolin/io/test_shapenet.py]: Ah, I was trying to run some tests locally. Changed now.

**Review by Caenorst — APPROVED** [review]
LGTM! Thanks! :)

**Pull request merged**